templates/jobs/job_edit.html

```diff
@@ -19,6 +19,13 @@
             <div class="panel panel-feature-blue">
                 <div class="panel-heading">
                     <h2>New offer</h2>
+                    <p>
+                        Adding a job offer on Django Girls website is free. Yay! Our website is monthly visited by XXX beginner developers and we can help you reach them with your offer. However, all the offers listed here need to meet following requirements:
+                        Offer needs to be beginner-friendly — it means no experience required. 
+                        Offer can’t discriminate any gender.
+                        You need to show us, somehow, that you care about diversity. We don’t want to recommend our friends to work in places that don’t value this.
```

Comment:
You need to show us that you care about improving diversity.  We don’t want to recommend to our friends that they work in places that don’t value this.